examples/src/osmo_cl.rs

```diff
@@ -1,5 +1,10 @@
 use std::{error::Error, str::FromStr, time::Duration};
 
+////////////////////////////////////////////
+// DECLARE TEST ENVIRONMENT CONFIGURATION //
```

Comment:
not sure if it's necessary to highlight the imports, feels pretty self explanatory